### DISCUSSION TITLE
Blog comment on Discover Healthsearch Unlocking Health with Semantic Search ✨

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1005] [Docs/blog comments]
# Blog comment on Discover Healthsearch Unlocking Health with Semantic Search ✨

Learn about our latest open source demo and how we used Semantic and Generative Search to improve access to health

https://weaviate.io/blog/healthsearch-demo

**Comment by obonyojimmy** [comment]
Fantastic 😍
